.formatter.exs

```diff
@@ -0,0 +1,4 @@
+# Used by "mix format"
+[
+  inputs: ["{mix,.formatter}.exs", "{config,lib,test}/**/*.{ex,exs}"]
```

Comment:
May want to `import_deps: [:ecto]` here.

Reply:
@BenMorganIO Done.